api/v1/errors/SqlError.js

```diff
@@ -0,0 +1,28 @@
+function checkError(err) {
```

Comment:
It doesn't throw an error though, at least, it doesn't get captured properly in my opinion. So without checking to see if SQL threw an error, it seems like there is no error at all, which I'm not sure if want to do.